VaticanBAV/et/BAVet122.xml

```diff
@@ -0,0 +1,453 @@
+                              ተድላሆሙ፡ ይረሲ፡ ለእለ፤ <sic>ይትሜጠው፡</sic> ንሥአተ፡ ቅዱስ፤ ምስጢር፡ ዘሥጋሁ፡ ወደሙ፡ ለክርስቶስ፡ ዘኵሎ፤ ይእኅዝ፤ እግዚአብሔር፤ አምላክነ፤ 
+                              ይ<supplied reason="explanation">ብል</supplied>፡ ዲ<supplied reason="explanation">ያቆን</supplied>፡ ጸልዩ። ይ<supplied reason="explanation">ብል</supplied>፡ 
+                              ካ<supplied reason="explanation">ህ</supplied>፡ ዘኵሎ፡ ትእኅዝ፡ ንሥአተ፡ ቅዱስ፡ ምሥጢር፤ ጽንዓተ፡ ለነ፡ ተሀበነ፡ ወኢመነሂ፡ 
+                              እምው<surplus>እ</surplus>ስቴትነ፤ <choice><sic>ኢትርስሕ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ኢታርስሕ፡</corr></choice> አላ፡ 
+                              ኵሎ፡ ባርክ፡ በክርስቶስ፡ በ፩ ወልድከ፡ በል፨
+                           </incipit>               
+                           <explicit xml:lang="gez">
+                              እምኵሉ፡ ምግባረ፡ እኩይ፤ አግኅሠነ፡ ውስተ፡ ኵሉ፤ ምግባረ፡ ሠናይ፡ ይደምረነ፨ ቡሩክ፡ ዘወሀበነ፤ ሥጋሁ፡ ቅዱሰ፤ ወደሞ፤ ክቡረ፡ ጸጋ፡ ነሣዕነ፤ ወሕይወተ፡ 
+                              ረከበነ፡ <sic>በኃ<supplied reason="omitted">ይ</supplied>ል፡</sic> መስቀሉ፡ ለኢየሱስ፡ ክርስቶስ፤ ኪያከ፡ <add place="above">እግዚኦ፡</add> 
+                              <sic>ነአኵቶ፤</sic> ነሢአነ፤ ጸጋ፤ ዘእመንፈስ፡ ቅዱስ። 
+                           </explicit>
+                           <note>The name of the owner, <persName role="owner" ref="PRS14614GabraMa">Gabra Māryām</persName>, is mentioned in the supplication 
+                              formula on <locus target="#147v"/>: 
+                              <q xml:lang="gez">ኦእግዚኦ፤ እቀበኒ፤ ወሠውረኒ፤ ሊተ፡ ለገብርከ፡ <persName role="owner" ref="PRS14614GabraMa">ገብረ፡ ማርያም፡</persName></q>.
+                           </note>
+                        </msItem>                     
+                     </msItem>      
+                     <colophon xml:lang="am" xml:id="coloph1">
+                        <locus target="#149v"/>
+                        ተፈጸመ፡ ሥርዓተ፤ ቅዳሴ፤ በዘመነ፤ ማርቆስ፡ በወርኃ፤ ኅዳር፤ <date calendar="ethiopian" when-custom="1905">፲፱፻፭ አመተ፤ ምሕረት፨</date> 
+                        ጸሕፊሁ፡ <persName role="scribe" ref="PRS14615Maqoyi"><roleName type="title">አለቃ፡</roleName> መቆዪ</persName>፨                       
+                        <note>
+                           The colophon states that the manuscript was written in the month of <foreign xml:lang="gez">Ḫədār</foreign> of the year of mercy 
+                           <date calendar="ethiopian" when-custom="1905">1905 EC</date>, i.e. 
+                           <date>1913 AD</date>. The colophon also mentions the name of the scribe
+                           <persName role="scribe" ref="PRS14615Maqoyi"><roleName type="title">ʾAlaqā</roleName> Maqoyi</persName>.
+                        </note>
+                     </colophon>
+                  </msItem>
+               </msContents>
+          
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <watermark>
+                           </watermark>
```

Comment:
could you delete watermark? I know it's copied from elsewhere, But I think its presence might be confusing in a parchment codex